### PR TITLE
Standardization in the new entry process + change in name field

### LINKED ISSUE #36: Change Name field entry
As mentioned [here](https://github.com/btcpayserver/directory.btcpayserver.org/pull/33#discussion_r411650224) `Name` field should be changed to something that reflects that better to avoid confusion, perhaps `Company/Project Name` would be better, but then again there are individuals as well, so field name can get messy, if someone has better suggestions how to handle this, please suggest.

### LINKED ISSUE #42: discrepancy in Type in list.js and submission form results on GitHub
## Plural vs Singular 
When user submit a submission over the website form, the type is `non-profit`, `host`, etc.
However in lists.js it's in plural `non-profits`, `hosts`.

While this is something a regular contributor can get used to, it may cause some issues and back and forth when someone tries to tackle an issue, so I suggest we standardize it.

@joerlop Perhaps there's a logic in here that I'm missing? I don't think it's a big deal and if there's logical explanation, I'm fine with it, we can see if it confuses people.

## Order or fields
Second, the type field in issue submission is 3rd
- Name 
- URL
- **Type**
- Description

Where as in submission form it's 4th

- Name 
- URL
- Description
- **Type**

Again this is not a big deal but it's kinda logical to have them in order so people can fill in data in the order when they're watching an issue.

### PR DESCRIPTION
**Pull request by Zaxounette**
**Standardization in the new entry process.**

- Added a comment in the list.js file to describe the inputs that need to be used in that file.

- Inverted "type" and "description" elements so that the printed order be "type" in 3rd" and "description" in 4th, to be consistent with all other orders. (newentry.js)
- Edited item list in newentry.js so that printed inputs in issues be consistent with inputs that need to be listed in list.js

Relates and closes #42 

- Changed the name field in newentry.js as to indicate that individuals can also submit, not just "organisations".

Relates and closes #36 

Sorry for the reverts, I got mixed up.

### CODE REVIEW

**Comment by Zaxounette** [comment]
Please wait before merge.

**Comment by Zaxounette** [comment]
@pavlenex This is a ping :) 

I don't know who else could review this, sorry Pav :p

**Comment by Zaxounette** [comment]
@joerlop @dennisreimann

**Comment by pavlenex** [comment]
Have you tested this locally? @Zaxounette

**Comment by Zaxounette** [comment]
Ahhhhhhhhhhhhhhhhhhh, no I have not.

The only string modification that is code related is in newentry.js, line 64.

All the other changes are not code related and are just syntaxe and such.

File: directory/src/components/NewEntry/NewEntry.js [hunk 1]
```diff
@@ -119,33 +119,33 @@ class NewEntry extends React.Component {
...
-                  <option value="Merchant - Holiday and Travel">Merchant - Holiday and Travel</option>
-                  <option value="Merchant - Jewelry">Merchant - Jewelry</option>
-                  <option value="Merchant - Pets">Merchant - Pets</option>
-                  <option value="Merchant - Services">Merchant - Services</option>
-                  <option value="Merchant - Software and Video Games">Merchant - Software and Video Games</option>
-                  <option value="Merchant - Sports">Merchant - Sports</option>
-                  <option value="Merchant - Streaming">Merchant - Streaming</option>
-                  <option value="Merchant - Tools">Merchant - Tools</option>
+                  <option value="hosts">Host</option>
+                  <option value="non-profits">Non-ptofits</option>
```
Comment (dennisreimann) [inline directory/src/components/NewEntry/NewEntry.js:126]: The title has the typo that was already fixed on master (2804beb84435792fc941bc8b6d1db435ca6e781c). Can you rebase or fix it manually?

Comment (Zaxounette) [inline directory/src/components/NewEntry/NewEntry.js:126]:
```suggestion
                  <option value="non-profits">Non-profits</option>
```

Comment (Zaxounette) [reply directory/src/components/NewEntry/NewEntry.js]: Fixed manually :)

**Pull request merged**